bin/reth/src/lib.rs

```diff
@@ -4,11 +4,15 @@
     no_crate_inject,
     attr(deny(warnings, rust_2018_idioms), allow(dead_code, unused_variables))
 ))]
+// TODO remove later
```

Comment:
what for is this?

Reply:
Removed. I added stub usage of the jwt resolution logic